Jenkinsfile

```diff
@@ -106,6 +106,7 @@ node {
           cp \"$AKAMAI_HOST_KEY\" ~/.ssh/known_hosts
           chmod 600 ~/.ssh/known_hosts
           rsync -arv -e \"ssh -2\" *.yml sshacs@cloud-unprotected.upload.akamai.com:${AKAMAI_APP_PATH}
+          rsync -arv -e \"ssh -2\" ./chrome/*.json sshacs@cloud-unprotected.upload.akamai.com:${AKAMAI_APP_PATH}/chrome
```

Comment:
@Hyperkid123 I see that we will be still be uploading the main yml file. Is the plan to support that for some deprecation period and then eventually get rid of it? 

Having a deprecation period would give me time to update the smoke tests for the new split config.

Reply:
Furthermore, I think we need to backup these JSON files in an analogous way to https://github.com/RedHatInsights/cloud-services-config/pull/666/files#diff-e6ffa5dc854b843b3ee3c3c28f8eae2f436c2df2b1ca299cca1fa5982e390cf8R46

Reply:
Currently, the `main.yml` is still required for the akamai/fakamai to host some static asses. Those will be required until every app migrated over chrome 2.

We don't know when it's gonna happen but we will start pushing the migration a bit more now. Hopefully, I will be able to give you a precise date soon.